### PR TITLE
Fix ordering of example and description text

### PR DESCRIPTION
**Pull request by prasadu-microsoft**
The description text for the examples referred to 'following..' while the example it referenced preceded it. I move the description above each example code. Also fixed code highlighting and the link to Get-ChildItem.

Version(s) of document impacted
------------------------------
- [ ] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/Microsoft.PowerShell.Security/Protect-CmsMessage.md](https://github.com/prasadu-microsoft/PowerShell-Docs/blob/patch-1/reference/5.1/Microsoft.PowerShell.Security/Protect-CmsMessage.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C19%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201803192117029536-2190%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4AWYSH5%2FDubJiWkgsRE6cd9MtVl6zUkXGUtCaJ3u8Uc%3D&st=2018-03-19T21%3A21%3A11Z&se=2018-04-19T21%3A26%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/5.1/Microsoft.PowerShell.Security/Protect-CmsMessage.md [hunk 1]
```diff
@@ -16,18 +16,18 @@ Encrypts content by using the Cryptographic Message Syntax format.
 ## SYNTAX
 
 ### ByContent (Default)
-```
+```PowerShell
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Security/Protect-CmsMessage.md]: Syntax blocks should not have code language labels. They are not valid PowerShell. See https://github.com/PowerShell/PowerShell-Docs/blob/staging/contributing/FORMATTING-CODE.md#syntax-block

File: reference/5.1/Microsoft.PowerShell.Security/Protect-CmsMessage.md [hunk 2]
```diff
@@ -79,24 +83,24 @@ ValidityPeriodUnits = "1000"
...
 
-### Example 3: View document encryption certificates
+```PowerShell
+PS C:\> $Protected = "Hello World" | Protect-CmsMessage -To "*youralias@emailaddress.com*"
 ```
+
+### Example 3: View document encryption certificates
+
+```PowerShell
 PS C:\> 58 [Cert:\currentuser\my]
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Security/Protect-CmsMessage.md]: This example does not make sense and it does not work. Can you fix this?

File: reference/5.1/Microsoft.PowerShell.Security/Protect-CmsMessage.md [hunk 3]
```diff
@@ -79,24 +83,24 @@ ValidityPeriodUnits = "1000"
...
-PS C:\> $Protected = "Hello World" | Protect-CmsMessage -To "*youralias@emailaddress.com*"
-```
 
 In the following example, you encrypt a message, Hello World, by saving the message in a variable, and then piping it to the **Protect-CmsMessage** cmdlet.
+
 The *To* parameter uses the value of the Subject line in the certificate.
 
-### Example 3: View document encryption certificates
+```PowerShell
+PS C:\> $Protected = "Hello World" | Protect-CmsMessage -To "*youralias@emailaddress.com*"
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Security/Protect-CmsMessage.md]: Remove the prompt strings from the examples. See https://github.com/PowerShell/PowerShell-Docs/blob/staging/contributing/FORMATTING-CODE.md#general-guidelines

**Comment by sdwheeler** [comment]
No response from contributor. Closing.

**Pull request closed without merging**